File: dataset/output/GEMINI_gemini-1.5-pro-001/Task146_GEMINI_gemini-1.5-pro-001.cpp
#include <vector>
#include <numeric>

int findMissingNumber(std::vector<int> arr) {
  int n = arr.size() + 1;
  int expectedSum = n * (n + 1) / 2;
  int actualSum = std::accumulate(arr.begin(), arr.end(), 0);
  return expectedSum - actualSum;
}

File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task146_GEMINI_gemini-1.5-pro-001.java
import java.util.Arrays;

public class FindMissingNumber {

    public static int findMissingNumber(int[] arr) {
        int n = arr.length + 1;
        int expectedSum = n * (n + 1) / 2;
        int actualSum = Arrays.stream(arr).sum();
        return expectedSum - actualSum;
    }

    public static void main(String[] args) {
        int[] arr = {1, 2, 4, 5, 6}; 
        int missingNumber = findMissingNumber(arr);
        System.out.println("The missing number is: " + missingNumber);
    }
}